modules/host/negotiatedownload.go

```diff
@@ -93,7 +35,7 @@ func (h *Host) managedDownloadIteration(conn net.Conn, so *storageObligation) er
 	// The renter will either accept or reject the host's settings.
 	err = modules.ReadNegotiationAcceptance(conn)
 	if err != nil {
-		return err
+		return extendErr("renter rejected host settings: ", ErrorCommunication(err.Error()))
```

Comment:
at first I thought this would result in:
`renter rejected host settings: communication error: foo`
but after rereading `extendErr` I see it's actually:
`communication error: renter rejected host settings: foo`
isn't this the same as:
`ErrorCommunication(fmt.Sprint("renter rejected host settings:", err))`
I suppose the advantage of `extendErr` is that if you extend an error multiple times, it will retain the same type. You'd need to be aware of this, though. For example, in the code above, if `ReadNegotiationAcceptance` returned `ErrorConnection`, it would be overwritten by `ErrorCommunication`.